R/chrome.R

```diff
@@ -71,14 +78,20 @@ find_chrome <- function() {
 }
 
 
-launch_chrome <- function(path = find_chrome(), args = character(0)) {
+launch_chrome <- function(path = find_chrome(), use_srgb_color_profile = TRUE, args = character(0)) {
```

Comment:
Now that I think about this some more, I don't think we even need to make this optional -- we might as well always set it.